driver.py

```diff
@@ -298,6 +298,121 @@ def throughput_scaling(self,
             # stop database after this set of permutations have finished
             exp.stop(db)
 
+    def aggregation_scaling(self,
+                            exp_name="aggregation-scaling",
+                            launcher="auto",
+                            run_db_as_batch=True,
+                            batch_args={},
+                            db_hosts=[],
+                            db_nodes=[12],
+                            db_cpus=36,
+                            db_port=6780,
+                            net_ifname="ipogif0",
+                            clients_per_node=[32],
+                            client_nodes=[128, 256, 512],
+                            iterations=20,
+                            tensor_bytes=[1024, 8192, 16384, 32769, 65538,
+                                          131076, 262152, 524304, 1024000,
+                                          2048000],
+                            tensors_per_dataset=[1,2,4],
+                            client_threads=[1,2,4,8,16,32]):
+
+        """Run the data aggregation scaling tests
```

Comment:
I'd maybe just mention real quick that the actual tests that will be run are a permutation of: client_nodes, clients_per_node, tensor_bytes, tensors_per_dataset, client_threads.